### ISSUE TITLE
Change the blue folder icons to orange

### ISSUE THREAD

**Issue by anasik** [btd1337/Stupendously-Hot-Charmander#15]
![image](https://cloud.githubusercontent.com/assets/2943955/21582733/d7556732-d085-11e6-8317-2c4b6424b5c6.png)
While this is aesthetic, we mustn't forget that the charmander project does not seek to change the color-scheme. Which is why, I'd much rather have:
![image](https://cloud.githubusercontent.com/assets/2943955/21582750/8d11eb2c-d086-11e6-8bab-44510e8fdcf8.png)

**Comment by Alzemand** [comment]
We can try like this: https://github.com/Alzemand/Ambience-Minimal-Icons

**Comment by rocketinventor** [comment]
@anasik Where does it say that?

**Comment by btd1337** [comment]
Alzemand vamos manter a estrutura atual. Basta trocar a cor das pastas. Tem esses arquivos em svg no projeto do Arc Icons no Github.

**Comment by Alzemand** [comment]
Sim. Os ícones estão bons, eu até gostei desse azul, pra ser sincero, mas podemos manter duas opções

**Comment by btd1337** [comment]
⁣Pode fazer isso?

Enviado por BlueMail ​

Em 5 de jan de 2017 19:08, em 19:08, Edilson Alzemand Sigmaringa Junior <notifications@github.com> escreveu:

**Comment by rocketinventor** [comment]
@Alzemand Who's going to maintain two different icon sets?

**Comment by Alzemand** [comment]
Posso fazer sim, yes you can 

Manteremos um tema de icone padrão, mas você poderá alterar no unity tweak tool

We will keep a standard icon theme, but you can change it in the unity tweak tool

**Comment by btd1337** [comment]
Implemented in https://github.com/btd1337/Stupendously-Hot-Charmander/commit/1c6e47414bfc360a08d1f013f38af658c733be7d

**Comment by anasik** [comment]
I'm not sure if it's working as intended. See, Only when I zoomed and scaled the icons to the 3rd biggest size, I got to see this:
![image](https://cloud.githubusercontent.com/assets/2943955/21748427/0670b258-d5a7-11e6-889e-ad4473bfa7db.png)

But at any other icon-size, it looks like this:
![image](https://cloud.githubusercontent.com/assets/2943955/21748430/1fbae0b2-d5a7-11e6-893f-24f78b511b5c.png)
![image](https://cloud.githubusercontent.com/assets/2943955/21748433/36e56f32-d5a7-11e6-83c0-48a96b08ec1c.png)
![image](https://cloud.githubusercontent.com/assets/2943955/21748436/46453cfa-d5a7-11e6-81c3-992ee523b601.png)

**Comment by btd1337** [comment]
Thanks for letting me know. We'll check.

**Comment by btd1337** [comment]
Hey guy, this folders yellow are from your another theme?

**Comment by anasik** [comment]
No... They are what I see when I apply charmander..

**Comment by anasik** [comment]
Apparently, they are part of numix.
